pmlc/dialect/tile/tests/legalize-to-stripe.mlir

```diff
@@ -0,0 +1,145 @@
+// RUN: pmlc-opt -tile-legalize-to-stripe %s | FileCheck %s --dump-input-on-failure
+
+func @eltwise_add(
+  %arg0: tensor<10x20x!eltwise.fp32>, 
+  %arg1: tensor<10x20x!eltwise.fp32>
+) -> tensor<10x20x!eltwise.fp32> {
+  %0 = "eltwise.add"(%arg1, %arg0) {type = !eltwise.fp32} : (
+    tensor<10x20x!eltwise.fp32>, 
+    tensor<10x20x!eltwise.fp32>
+  ) -> tensor<10x20x!eltwise.fp32>
+  return %0 : tensor<10x20x!eltwise.fp32>
+}
+
```

Comment:
Same comment as before. Please, try to make these tests more resilient to changes not related with the goal of this tests.

You can also use the `--split-input-file` flag with the separator `// -----` to separate multiple tests within the same file: https://github.com/tensorflow/mlir/blob/master/test/Transforms/loop-fusion.mlir#L11

Reply:
Yep, I know about the `--split-input-file`, thanks. However, this use case doesn't need it.